bindings/profilers/wall.cc

```diff
@@ -275,10 +307,16 @@ void SignalHandler::HandleProfilerSignal(int sig,
     return;
   }
 
+  int64_t cpu_time = 0;
+  if (prof->collectCpuTime()) {
+    timespec ts;
+    clock_gettime(CLOCK_THREAD_CPUTIME_ID, &ts);
+    cpu_time = timespec_to_duration(ts).count();
```

Comment:
Any reason we shouldn't use 
```suggestion
    cpu_time = currentThreadCpuTime();
```
here? I guess you can mark the method `inline` if you don't want to incur a method call cost for some reason.

Reply:
I might have added `currentThreadCpuTime()` later on and forgot to change all the uses.
Fixed.